jabgui/src/main/java/org/jabref/migrations/PreferencesMigrations.java

```diff
@@ -612,6 +613,48 @@ static void addICORERankingFieldToGeneralTab(GuiPreferences preferences) {
      * The tab "Comments" is hard coded using {@link CommentsTab} since v5.10 (and thus hard-wired in {@link org.jabref.gui.entryeditor.EntryEditor#createTabs()}.
      * Thus, the configuration ih the preferences is obsolete
      */
+
+    static void addEprintTypeFieldToGeneralTab(GuiPreferences preferences) {
```

Comment:
This method is very similar to `addICORERankingFieldToGeneralTab`. They should be unified somehow. - Maybe, `oldDefaultGeneralFields`. This can be a `Set` and if the current field set matches any of that set, then reset to the default.

I think, your `null` check is better than the one of the ICORE method.